cloudify_cli/tests/test_bootstrap.py

```diff
@@ -99,3 +100,70 @@ def test_creation_validation_no_docker(self):
             self.assertIn(
                 '"docker" must be a non-empty dictionary property under '
                 '"cloudify_packages"', ex.message)
+
+    def test_ssl_configuration_without_cert_path(self):
+        configurations = {
+            constants.SLL_ENABLED_PROPERTY_NAME: True,
+            constants.CERTIFICATE_PATH_PROPERTY_NAME: '',
+            constants.PRIVATE_KEY_PROPERTY_NAME: ''
+        }
+        try:
+            tasks._handle_ssl_configuration(ssl_configuration=configurations)
+            self.fail('NonRecoverableError expected.')
+        except NonRecoverableError as e:
+            self.assertIn(
+                'SSL is enabled => certificate path must be provided',
+                e.message)
+
+    def test_ssl_configuration_wrong_cert_path(self):
+        configurations = {
+            constants.SLL_ENABLED_PROPERTY_NAME: True,
+            constants.CERTIFICATE_PATH_PROPERTY_NAME: 'wrong-path',
+            constants.PRIVATE_KEY_PROPERTY_NAME: ''
+        }
+        try:
+            tasks._handle_ssl_configuration(ssl_configuration=configurations)
+            self.fail('NonRecoverableError expected.')
+        except NonRecoverableError as e:
+            self.assertIn('The certificate path [wrong-path] does not exist',
+                          e.message)
+
+    def test_ssl_configuration_without_key_path(self):
+        this_dir = os.path.dirname(os.path.dirname(__file__))
+        cert_path = os.path.join(this_dir, 'cert.file')
+        with open(cert_path, 'a+'):
```

Comment:
since you're not reading or changing the file created in "cert_path", you can just:
open (cert_path, 'a+').close()
without the "with" and all the rest